backend/internal/usecase/sketch_usecase.go

```diff
@@ -16,8 +16,8 @@ func NewSketchUsecase(pr repository.SketchRepository) SketchUsecase {
 	}
 }
 
-func (u *SketchUsecase) CreateSketch(file *multipart.File) error {
-	return u.sketchRepository.CreateSketch(file)
+func (u *SketchUsecase) CreateSketch(file *multipart.File, userid int) error {
```

Comment:
[NITS]
idは省略後なので、大文字で。これgolangらしさ。
```suggestion
func (u *SketchUsecase) CreateSketch(file *multipart.File, userID int) error {
```